lib/atm.rb

```diff
@@ -3,27 +3,27 @@
 class Atm
   attr_accessor :funds
   STANDARD_PIN = 1234
-  STANDARD_EXP_DATE = "10/17"
+  STANDARD_EXP_DATE = "10/17".freeze
 
-   def initialize
+  def initialize
     @funds = 1000
-   end
+  end
 
-   def withdraw(amount, pin, expiry, account)
-      case
-      when negative_value(amount) then return_error_message(:negative_amount)
-      when !check_pin(pin, account.pin) then return_error_message(:wrong_pin)
-      when check_expiry(expiry, account.exp_date) then return_error_message(:card_expired)
-      when !sufficient_fund_in_atm(amount) then return_error_message (:no_sufficient_fund)
-      when amount % 5 != 0 then return_error_message (:non_rounded_amount)
-
-      else
-        do_transaction(amount, account)
-        respons = { status: true,
-          message: :success,
-          date: Date.today.strftime("%F"),
-          amount: amount,
-          bills: do_bill_count(amount)}
+  def withdraw(amount, pin, expiry, account)
+    case
+    when negative_value?(amount) then return_error_message(:negative_amount)
+    when !correct_pin?(pin, account.pin) then return_error_message(:wrong_pin)
+    when card_expired?(expiry, account.exp_date) then return_error_message(:card_expired)
+    when !suffient_funds_in_account?(amount, account) then return_error_message(:account_balance_too_low)
+    when !sufficient_fund_in_atm?(amount) then return_error_message :no_sufficient_fund
```

Comment:
Metrics/LineLength: Line is too long. [87/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)